test/gui/tst_syncing/test.feature

```diff
@@ -329,4 +330,97 @@ Feature: Syncing files
         And as "Alice" file "testavatar.jpeg" should exist on the server
         And as "Alice" file "testaudio.mp3" should exist on the server
         And as "Alice" file "test_video.mp4" should exist on the server
-        And as "Alice" file "simple.txt" should exist on the server
\ No newline at end of file
+        And as "Alice" file "simple.txt" should exist on the server
+  
+
+    Scenario Outline: File with long name can be synced
+        Given user "Alice" has set up a client with default settings
+        # the length of file name is 224
+        When user "Alice" creates a file "<filename>" with the following content inside the sync folder
+            """
+            test content
+            """
+        And the user waits for file "<filename>" to be synced
+        Then as "Alice" file "<filename>" should exist on the server
+        Examples:
+            | filename                                                                                                                                                                                                                        |
+            | sdkdfjsdfuidjfkdsjfksdjfksdjfksdjfksdjfksdjfkdsjfksdjfkdsjfkdsfjlsdkfjsdkjflksdjfklsdjfksdjfkdsjfkldsjfkldsjfkdsjfksdjfksdjfklsdjfklsdjflksdjflksdjfklsdjfklsdjfksdjfksdjfksdjfksdjfksdfjskdfjksdjfksdjfksdjfksdjfksdwwwwww.txt |
```

Comment:
can you use a little bit more readable name?

Reply:
In https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiSharePublicLink1/createPublicLinkShare.feature we have:
`aquickbrownfoxjumpsoveraverylazydogaquickbrownfoxjumpsoveralazydog.txt`

You can write a whole sentence that is readable. You can even explain in the file name `thisIsAVeryLongFileNameToCheckThatItWorks0123456789.txt`